### PR TITLE
Task-65-Search-Panel-with-bookmark-css-tweaks

### PR DESCRIPTION
**Pull request by hnbmunoz**

### CODE REVIEW

File: react-client/src/Components/App.js [hunk 1]
```diff
@@ -237,26 +238,25 @@ function App() {
...
         const isNotBtnDisable = !domUtils.ancestorExistsByClassName(e.target, "dom-switch");
         const isNotSelectedDomFromBookmark = !domUtils.ancestorExistsByClassName(e.target, 'selected-dom');
+        const isNotSelectedFromSearch = !domUtils.ancestorExistsByClassName(e.target, 'search-panel');
+
         
-        if (isNotDomInfoComponent && isNotBtnDisable && e.target.nodeName !== "HTML" && isNotSelectedDomFromBookmark && !focusMode) {
+        if (isNotDomInfoComponent && isNotBtnDisable && e.target.nodeName !== "HTML" && isNotSelectedDomFromBookmark && isNotSelectedFromSearch && !focusMode) {
           const domType = e.target.nodeName?.toLowerCase();
-          await setDomLeanDetails({
+           setDomLeanDetails({
```
Comment (reciosonny) [inline react-client/src/Components/App.js]: why await is removed?

Comment (hnbmunoz) [reply react-client/src/Components/App.js]: already returned unintentionally removed

**Comment by reciosonny** [comment]
Okay will check and run the app later. Thanks.

**Review by reciosonny — APPROVED** [review]

**Comment by reciosonny** [comment]
Approved.

**Pull request merged**